### PR TITLE
make config name and number of workers configurable from command line

### PR DESCRIPTION
**Pull request by johnwlambert**

### CODE REVIEW

File: gtsfm/runner/run_scene_optimizer.py [hunk 1]
```diff
@@ -55,6 +55,24 @@ def run_scene_optimizer(args) -> None:
         default=20,
         help="maximum number of consecutive frames to consider for matching/co-visibility",
     )
+    parser.add_argument(
+        "--n_workers",
```
Comment (akshay-krishnan) [inline gtsfm/runner/run_scene_optimizer.py:59]: I think num_workers sounds better, but up to you.

**Review by akshay-krishnan — APPROVED** [review]

File: gtsfm/runner/run_scene_optimizer.py (hunk 1 above)
Comment (johnwlambert) [reply gtsfm/runner/run_scene_optimizer.py:59]: Thanks Akshay. i like that -- just updated it.

File: gtsfm/runner/run_scene_optimizer.py [hunk 2]
```diff
@@ -55,6 +55,24 @@ def run_scene_optimizer(args) -> None:
...
+        "--threads_per_worker",
+        type=int,
+        default=1,
+        help="Number of threads per each worker",
+    )
+    parser.add_argument(
+        "--config_name",
+        type=str,
+        default="deep_front_end.yaml",
+        help="Choose default_lund_door_set1_config.yaml or deep_front_end.yaml",
```
Comment (ayushbaid) [inline gtsfm/runner/run_scene_optimizer.py:74]: maybe time to rename `default_lund_door_set1_config`?

Comment (johnwlambert) [reply gtsfm/runner/run_scene_optimizer.py]: I went ahead and changed the name to `sift_front_end.yaml` @ayushbaid -- let me know how that looks

**Review by akshay-krishnan — APPROVED** [review]

**Pull request merged**